evm/auxiliary/user_account.py

```diff
@@ -0,0 +1,181 @@
+from eth_keys import keys
+
+from evm.vm.forks.sharding.transactions import (
+    ShardingTransaction,
+)
+
+from evm.validation import (
+    validate_uint256,
+    validate_canonical_address,
+    validate_is_bytes,
+)
+
+from evm.utils.hexadecimal import (
+    decode_hex,
+)
+from evm.utils.numeric import (
+    big_endian_to_int,
+    int_to_big_endian,
+)
+from evm.utils.padding import (
+    pad32,
+)
+
+
+USER_ACCOUNT_CODE = decode_hex(
+    "0x3f595260805903608020600052602060006080600060006001610bb8f1506001600051"
```

Comment:
Will you add a comment referencing the *source code* that produced this bytecode and how it can be reproduced/verified.

Reply:
Yes, this is not much more than a placeholder for now. Ideally I'd like to put the source code there directly and compile it in a test for comparison, but this will probably not work because Viper needs Python 3.6.

Reply:
#206 has the same issue, it's better not import Viper even in Python 3.6. 

A (little complicated) suggestion of verifying Viper code:
1. A command for compiling Viper source code to bytecode.
2. If someone want to update the code, they have to upload `source code` and `bytecode` (can be in separate file or assigned to a global variable in this case)
3. Install Viper in CI testing.
4. Write a test for auto-compiling the source code and then comparing the compiled bytecode with `evm.auxiliary.user_account.USER_ACCOUNT_CODE`.

By the way, @pipermerriam what do you think about upgrading to Python 3.6?

Reply:
Perhaps the cleanest solution would be to put the code plus tests in a separate repository. The repository would contain packages with the bytecode for different languages that can be imported into client implementations such as Py-EVM.

Reply:
> By the way, @pipermerriam what do you think about upgrading to Python 3.6?

I'm in favor: https://github.com/ethereum/py-trie/pull/18

Reply:
I'm indifferent to the actual solution for the bytecode creation as long as it's documented and independently verifiable.